engine/src/main/java/org/terasology/rendering/nui/contextMenu/ContextMenuBuilder.java

```diff
@@ -15,72 +15,53 @@
  */
 package org.terasology.rendering.nui.contextMenu;
 
-import com.google.common.base.Preconditions;
 import com.google.common.collect.Lists;
 import org.terasology.math.geom.Vector2i;
 import org.terasology.rendering.nui.NUIManager;
 import org.terasology.rendering.nui.databinding.Binding;
 import org.terasology.rendering.nui.databinding.ReadOnlyBinding;
 import org.terasology.rendering.nui.widgets.UIList;
-import org.terasology.rendering.nui.widgets.UpdateListener;
 
 import java.util.List;
 
+/**
+ * A builder class to initialize and display {@link ContextMenuScreen} instances.
+ * <p>
+ * Should be used in favor of manually creating the screen.
+ */
 public class ContextMenuBuilder {
```

Comment:
It's not a builder any more, but a utility class. I would call it ContextMenuUtil. , I suggest you update the description and add a private constructor with a comment in the constructor that it is a utility class.